src/bin/hq.rs

```diff
@@ -10,9 +10,12 @@ use hq_rs::{parse_filter, query, write};
 #[derive(Parser)]
 #[command(version, about)]
 struct Args {
-    #[clap(short, long, value_name = "FILE", help = "HCL file to read from")]
+    #[clap(short = 'f', long = "file", value_name = "FILE", help = "HCL file to read from")]
     file: Option<String>,
 
+    #[clap(short = 'i', long = "inline", help = "Write to HCL file inline instead of stdout (--file must also be set)", requires="file")]
+    inline: bool,
```

Comment:
This should be an option for the `write` command only, not defined at the top level like this

Reply:
tested it out...

```
➜ cp example.hcl copy.hcl

➜ hq -f copy.hcl '.fmt_block.first_formatted_field'
"fmt_value"

➜ hq -f copy.hcl -i '.fmt_block.first_formatted_field' write '"new_value"'

➜ hq -f copy.hcl '.fmt_block.first_formatted_field'
"new_value"
```

works great! just a little awkward to put `-i` so far left of `write`, when I'd expect it to be structured like:

```
hq -f copy.hcl '.fmt_block.first_formatted_field' write -i '"new_value"'
```